projects/cashmere-examples/src/lib/checkbox-indeterminate/checkbox-indeterminate-example.component.html

```diff
@@ -0,0 +1,10 @@
+<hc-checkbox [checked]="isChecked()" [indeterminate]="isIndeterminate()" (change)="parentClick()">
+    Parent Checkbox (indeterminate)
+</hc-checkbox>
+
+<div class="checkbox-group">
+    <hc-checkbox [checked]="childChecks[0]" (change)="childClick($event)" id="0">Child Checkbox</hc-checkbox>
```

Comment:
Although this works, I think it would be better to have our example use Angular Reactive Forms instead. (as it's more or less a best practice.  In this example, the model is changed by the UI, but you can't initialize the form with previously saved values.  IMHO it makes this example less compelling.